src/main/java/com/pretz/dyntreeserver/AuthController.java

```diff
@@ -18,23 +19,27 @@ public AuthController(UserService userService) {
         this.userService = userService;
     }
 
-    @RequestMapping(method = GET, path = "/auth")
-    public String login(@RequestParam(value = "name") String name, @RequestParam(value = "password") String password) {
-        User user = new User(name, password);
-        if (userService.checkIfUserExists(user)) {
-            return "User exists!"; //TODO to be changed into some json response for client (?)
-        } else {
-            return "No such user exists!"; //TODO to be changed into some error message to client
+    @RequestMapping(method = POST, path = "/auth")
+    public ResponseEntity<String> login(@RequestBody User user) {
+        boolean userExists = userService.validateUser(user);
+        if (userExists) {
+            //is it RESTful?
+            UserSession userSession = new UserSession(user);
```

Comment:
Is this stored somewhere? Usually either a JWT is sent back to the user as payload, or a session token is set in cookies.
How do you handle it here?

Reply:
(+ I think that it'd be better to use sth like [Spring Session](https://projects.spring.io/spring-session/) )

Reply:
It is a Session stub. I didn't know how to proceed yet. I will probably follow your suggestions, but I don't know if I want to use Spring Session or simply create some logic myself for training. ;)